utils/rtl_433/Makefile

```diff
@@ -0,0 +1,52 @@
+# 
+# Copyright (C) 2017 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rtl_433
+PKG_VERSION:=6531ea4
+PKG_RELEASE:=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/merbanan/rtl_433.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=6531ea48a7933ac8289724672059e54fd8aad8eb
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+CMAKE_INSTALL:=1
+
+PKG_LICENSE:=GPLv2
+PKG_LICENSE_FILES:=COPYING
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/rtl_433
+  TITLE:=Realtek RTL2832 based 433.92MHz generic data receiver
+  SECTION:=utils
+  CATEGORY:=Utilities
+  DEPENDS:=+librtlsdr +rtl-sdr +libtool-bin
+  URL:=https://github.com/merbanan/rtl_433
+  MAINTAINER:=Jasper Scholte <NightNL@outlook.com>
+endef
+
+define Package/rtl-sdr/description
+  rtl_433 turns your Realtek RTL2832 based DVB dongle into a 433.92MHz generic data receiver.
+endef
+
+TARGET_CFLAGS += $(FPIC)
+
+define Build/InstallDev
```

Comment:
Why is this needed?

Reply:
Indeed. It's not used as a library. I just copied some stuff from rtl-sdr. Sorry